eduVPN/ConnectionService.swift

```diff
@@ -294,7 +301,14 @@ class ConnectionService: NSObject {
             } else {
                 self.coolDown()
                 self.configURL = nil
-                handler(.failure(Error.helperRejected))
+                if(error == "harmfulConfiguration" ){
+                    print("Malicious command found: "+message)
+                    handler(.failure(Error.test))
```

Comment:
Use an associated value with a new error, say `case potentiallyHarmfulCommands([String])` consisting of an array of the comments that you found and use it to create the error message. Look for the examples using this elsewhere in the app.

Reply:
Ok